src/runtime/virtcontainers/kata_agent.go

```diff
@@ -309,13 +310,26 @@ func (k *kataAgent) longLiveConn() bool {
 	return k.keepConn
 }
 
+// agentLogLevel calculates the log level, LogLevel precedes the Debug flag.
+func agentLogLevel(config KataAgentConfig) string {
+	level := ""
+	if config.Debug {
```

Comment:
this means we still use `enable_debug` in agent config to control log_level?

Reply:
The current code uses `enable_debug` to control kata-agent's log level, too. This PR just makes the kata-agent log level configurable (instead of the current one that's always `debug`).
```
if config.Debug {
		params = append(params, Param{Key: "agent.log", Value: "debug"})
	}
```